packages/wrangler/src/pages/upload.tsx

```diff
@@ -43,6 +47,8 @@ export const Handler = async ({
 	directory,
 	outputManifestPath,
 }: UploadArgs) => {
+	console.log(MAX_ASSET_COUNT);
```

Comment:
is this `log` intentional?

Reply:
Nope, thanks. I was trying to do #2106 in this PR as well, but just decided to do it separately.